code/modules/uplink/uplink_items.dm

```diff
@@ -1461,17 +1450,6 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/ai_module/syndicate
 	cost = 4
 
-//SKYRAT EDIT BEGIN - Brainwash surgery no longer restricted
-/datum/uplink_item/device_tools/brainwash_disk
-	name = "Brainwashing Surgery Program"
-	desc = "A disk containing the procedure to perform a brainwashing surgery, allowing you to implant an objective onto a target. \
-	Insert into an Operating Console to enable the procedure."
-	item = /obj/item/disk/surgery/brainwashing
-	cost = 5
-//SKYRAT EDIT END
-
-//SKYRAT EDIT REMOVAL BEGIN - Remove Hypnostuff
-/*
 /datum/uplink_item/device_tools/hypnotic_flash
```

Comment:
The hypnotic devices would need to be confirmed to be working before we let them back in, they were removed due to literally not working.

Reply:
These work upstream, provided you're using the stock /tg/ code for these here.

Reply:
I think it might be related to something like ambitions, honestly, I don't know, but I remember them being disabled because they didn't work. If you could test them to confirm they do work, then we can have them no problem.